ChartsDemo/Swift/Demos/BubbleChartViewController.swift

```diff
@@ -104,7 +104,7 @@ class BubbleChartViewController: DemoBaseViewController {
         set3.setColor(ChartColorTemplates.colorful()[2], alpha: 0.5)
         set3.drawValuesEnabled = true
         
-        let data = BubbleChartData(dataSets: [set1, set2, set3])
+        let data = [set1, set2, set3] as BubbleChartData
```

Comment:
may I know which protocol method enables this convenient conversion?

Reply:
`ExpressibleByArrayLiteral`